### ISSUE TITLE
Script causes crashes on multiplayer

### ISSUE THREAD

**Issue by raven06a** [pet333r/pw-dev_script#54]
I was having problems with DCS crashing when joining multiplayer servers. Started 3 weeks ago and it has persisted. Everything is up to date, did a clean installation of DCS, repaired everything and then I moved the folders one by one back into the SavedGames folder. 

The only thing that makes DCS crash is the UFC script. I have tried removing it completely and all the residual files, reinstall, more than a few times. The only constant is once I remove the script everything works fine. 

[dcs.log](https://github.com/pet333r/pw-dev_script/files/15206447/dcs.log)

**Comment by InzaneSK** [comment]
I am having the exact same issue. Works in SP but crash in MP. Attached is my DCS.log crash file, that might help you out. 

[dcs.log](https://github.com/pet333r/pw-dev_script/files/15210512/dcs.log)

**Comment by raven06a** [comment]
Yes, I also had to completely remove the saved games folder and add each file one by one to make sure. There are residual files in Hooks and I think had some problems within the Config folder too.

**Comment by pet333r** [comment]
try deleting the "pw_nav-hook.lua" file from the Hooks folder

**Comment by domagojhack** [comment]
I can confirm that removing "pw_nav-hook.lua" allows connecting to mp servers.

**Comment by Mayhem93** [comment]
I can confirm this behavior with both the latest version and an older one (a few months old).

I found out that this script was the issue by removing everything (I had a backup) in the saved games/DCS folder and reinstalling the script.

Sadly enough the functionality to remove it using the Script Configurator does not work as it still leaves a script file behind (even though the export lua is empty).

**Comment by domagojhack** [comment]
Any news on this issue?

**Comment by DavidCB75** [comment]
I have this issue too. The only workaround I found is to load a flight offline, exit of the flight, and then I can join a multiplayer server (all this without exiting the game, because if you leave the game, you have to repeat the next time). I will looking forward to see this fixed, I mainly use DCS NAV and it is very useful to me with the Huey.

**Comment by Mayhem93** [comment]
> I have this issue too. The only workaround I found is to load a flight offline, exit of the flight, and then I can join a multiplayer server (all this without exiting the game, because if you leave the game, you have to repeat the next time). I will looking forward to see this fixed, I mainly use DCS NAV and it is very useful to me with the Huey.

This workaround just avoids the crash, unfortunately the App doesn't connect when trying on a MP server.

I have the latest 2024.05.23 script version and the latest 2.9.5.55300 game version.

**Comment by Mayhem93** [comment]
Hey guys, I think the issues has been finally fixed. I was able to join a MP server with this mod/script installed with no game crash. You just have to use the newer script configurator then update the script using it.
